app/controllers/blog_controller.rb

```diff
@@ -100,8 +100,12 @@ def blog_html_with_additions(absolute_path)
 
     html.sub!('</body>') { "#{helpers.ts_tag('comments')}</body>" }
 
-    if params[:action] == 'show'
```

Comment:
We now want to render Tailwind on the blog index page, as well, so that it can be used to style the logged-in header, which we will render there if a user is logged in.